floris/tools/visualization.py

```diff
@@ -71,27 +73,40 @@ def plot_turbines(
         ax.plot([x_0, x_1], [y_0, y_1], color=color)
 
 
-def plot_turbines_with_fi(fi: FlorisInterface, ax=None, color=None, yaw_angles=None):
+def plot_turbines_with_fi(
+    fi: FlorisInterface,
+    ax=None,
+    color=None,
+    wd=None,
+    yaw_angles=None,
+):
     """
     Wrapper function to plot turbines which extracts the data
     from a FLORIS interface object
 
     Args:
-        fi (:py:class:`floris.tools.flow_data.FlowData`):
-                FlowData object.
-        ax (:py:class:`matplotlib.pyplot.axes`): figure axes.
-        color (str, optional): Color to plot turbines
+        fi (:py:class:`floris.tools.floris_interface.FlorisIntreface`): FlorisIntreface object.
+        ax (:py:class:`matplotlib.pyplot.axes`): Figure axes. Defaults to None.
+        color (str, optional): Color to plot turbines. Defaults to None.
+        wd (list, optional): The wind direction to plot the turbines relative to. Defaults to None.
+        yaw_angles (NDArray, optional): The yaw angles for the turbines. Defaults to None.
     """
     if not ax:
         fig, ax = plt.subplots()
     if yaw_angles is None:
         yaw_angles = fi.floris.farm.yaw_angles
+    if wd is None:
+        wd = fi.floris.flow_field.wind_directions[0]
```

Comment:
This may be incorrect given the line in `plot_turbines` function call below that also extracts the first wind direction. Double check @bayc

Reply:
Oh I see now that this is only used to adjust the yaw angles. That's probably good to reflect in the doc string